src/AdminConsole/Pages/Account/Login.cshtml.cs

```diff
@@ -32,16 +39,34 @@ public async Task<IActionResult> OnPost(string email, string? returnUrl = null)
     {
         returnUrl = Url.Page("/Organization/Overview");
 
+        var user = await _signInManager.UserManager.FindByEmailAsync(email);
+
+        if (user == null)
+        {
+            return RedirectToPage();
+        }
+
+        var organization = await _dataService.GetOrganizationAsync(user.OrganizationId);
```

Comment:
1. If user is not found, abort and refresh the page.
2. If organization is not found related to the user that was found, abort and refresh the page (would never happen).
3. If magic links is disabled, set the correct status and refresh the page so we can see the error message.

Reply:
Step 3 also allows for account enumeration by identifying users that belong to orgs that have disabled magic links.

One suggestion would be to still send an email, but just have the email not be a magic link email.

Instead it would be informing the user that their org has disabled magic link sign ins and to contact their admin, a message similar to the one implemented on the page.

Reply:
@jrmccannon Check now, I am showing the same message as when magic links are sent. And it's sending the e-mail.